tlair/src/main/scala/at/forsyte/apalache/tla/lir/oper/ApalacheInternalOper.scala

```diff
@@ -0,0 +1,39 @@
+package at.forsyte.apalache.tla.lir.oper
+
+/**
+ * The operators defined in the module `__apalache_internals.tla`. The operators in this module are designed for the
+ * Apalache code and its internal TLA+ definitions. The users should not use these operators.
+ *
+ * @author
+ *   Igor Konnov
+ */
+abstract class ApalacheInternalOper extends TlaOper {
+  override def interpretation: Interpretation.Value = Interpretation.StandardLib
+}
```

Comment:
```suggestion
abstract class ApalacheInternalOper extends ApalacheOper
```

Reply:
Though arguably, you could just remove this and have the other operators directly extend `ApalacheOper `, since this class isn't doing anything.